### ISSUE TITLE
Profile Access Control

### ISSUE THREAD

**Issue by Gergling** [Gergling/crowdfile#2]
Any attributes on a profile can be updated by anybody, as long as they're logged in.

All profiles and their contents are publicly visible.

**Comment by GregoryCollett** [comment]
This makes life nice and easy. I guess that leaves defining what the profile actually contains? I assume you already have some idea of this.

**Comment by Gergling** [comment]
Good point. I wrote a theme to answer this: https://github.com/Gergling/crowdfile/issues/8

I'm thinking about somewhere to write comments, but frankly people have a plethora of other social media tools they can use for that.
